dim-core/src/scanner/mod.rs

```diff
@@ -41,11 +42,65 @@ use tracing::error;
 use tracing::info;
 use tracing::instrument;
 use tracing::warn;
-use walkdir::WalkDir;
 
 pub use error::Error;
 
-pub(super) static SUPPORTED_EXTS: &[&str] = &["mp4", "mkv", "avi", "webm"];
+pub(super) static SUPPORTED_EXTS: &[&str] = &[
```

Comment:
Oh boy, not a fan of how big this is now. Can we maybe move it to a file or a separate module and not inline it?

Reply:
I'm happy to move this elsewhere. If you have suggestions for where that file or module should be located and named let me know and I'll make the adjustment.

Reply:
We can always tidy it later, good-enough for now :)